app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -253,9 +279,17 @@ private void tick() {
                     .setThrowable(e)
                     .log();
         } finally {
-            recordCounter("create", error.isPresent() ? 0 : 1);
+            recordCounter("create", status instanceof Status.Success ? 0 : 1);
+        }
+        return status;
+    }
+
+    private void updateCache(final LocalDate start, final LocalDate end) {
+        LocalDate date = start;
+        while (!date.equals(end)) {
```

Comment:
As these are local dates, I believe they won't be equal if they have different locales (zones) even if adjusted it's the same point in time. Is this what we want for comparison? (related: see comment about zones above)

Reply:
If I understand the concern, it's not applicable here because LocalDate instances do not have timezone information - it's essentially just day/month/year

Reply:
(I'm treating all time in here as UTC which is why the zone offset information was dropped)